downstream/modules/platform/con-backup-aap.adoc

```diff
@@ -0,0 +1,17 @@
+[id="con-backup-aap_{context}"]
+
+= Back up your {PlatformNameShort} instance
+
+Back up an existing {PlatformNameShort} instance by running the `.setup.sh` script with the `backup_dir` flag, which saves the content and configuration of your current environment:
+
+.Procedure
```

Comment:
Why is there **Procedure** here and not in other sections before this where there are procedures?

Reply:
Along the same lines, some steps have periods at the end, and some don't.. Not sure the convention, but should be consistent.